app/main/forms/direct_award_forms.py

```diff
@@ -30,6 +30,13 @@ def options(self):
         return [{"label": label, "value": value} for value, label in self.choices]
 
 
+class DMBooleanField(BooleanField):
```

Comment:
Is this needed? It feels like we have this in utils now

Reply:
That PR didn't land yet :( https://github.com/alphagov/digitalmarketplace-utils/pull/400

Reply:
Aww, ok